brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
First of all, imho, stride should be in the same units as start and end.
So if start and end are times in milliseconds, I would expect stride to be the same.

Restricting it to be a multiple of time_step should suffice; thus no need for interpolation.